src/Core/Domain/Order/Command/IssuePartialRefundCommand.php

```diff
@@ -80,7 +79,7 @@ class IssuePartialRefundCommand
      * @param int $shippingCostRefund
      * @param bool $restockRefundedProducts
      * @param bool $generateCartRule
-     * @param int $taxMethod
+     * @param bool $taxMethod
```

Comment:
Are you sure taxMethod is an int? :thinking:

Reply:
it's indeed a bool, fixed !